inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,108 @@
+#Pseudocode may look like actual code...please ignore...
+get_pdb_ids <- function(target_seq, start_pos = 1 , end_pos = nchar(target_seq), convert = TRUE, silent = TRUE){
+  
+  #get packages
+  # install.packages("bio3d")
+  # install.packages("RCurl")
+  # install.packages("XML")
+  # install.packages("seqinr")
+  # library(bio3d)
+  # library(RCurl)
+  # library(XML)
+  # library(seqinr)
+  
+  
+  
+  #convert sequence into AA 
+  if(convert){
+    #translate needs a vector of single chars, not a string
+    AA_seq <- paste(translate(s2c(target_seq)), collapse='')
+    end_pos <- nchar(AA_seq)
+  }else{ AA_seq <- target_seq }
+ 
+  blast_output <- blast.pdb(AA_seq)
+  
+  blast_output <- blast_output$hit.tbl
+  blast_output <- blast_output[blast_output$identity > 40, ]
```

Comment:
Don't you need to align, then check **where** you actually have 100% ID? You might have 90% overall, but including malaligned fringes...